Bucketeer/Sources/Public/BKTError.swift

```diff
@@ -53,11 +59,17 @@ extension BKTError {
                     self = .badRequest(message: errorResponse?.error.message ?? "BadRequest error")
                 case 401:
                     self = .unauthorized(message: errorResponse?.error.message ?? "Unauthorized error")
+                case 403:
+                    self = .forbidden(message: errorResponse?.error.message ?? "Forbidden error")
                 case 404:
                     self = .featureNotFound(message: errorResponse?.error.message ?? "NotFound error")
```

Comment:
I couldn't find the `NotFoundErrorMetricsEvent` in the MetricsEventData.swift for 499.

Reply:
Added handlings https://github.com/ca-dp/bucketeer-ios-sdk/pull/92/commits/cde633891b0ccb15c434164160feec2bbe3da073